katalog/node-exporter/MAINTENANCE.md

```diff
@@ -16,3 +16,12 @@ Replace `KUBE_PROMETHEUS_RELEASE` with the current upstream release.
 3. Sync the new image to our registry in the [`monitoring` images.yaml file fury-distribution-container-image-sync repository](https://github.com/sighupio/fury-distribution-container-image-sync/blob/main/modules/monitoring/images.yml).
 
 4. Update the `kustomization.yaml` file with the new image.
+
+## Customizations
+
+We added the `GOMAXPROCS=1` environment variable to limit the go rutines to 1 processor because node-exporter was getting throttled hard by Kubernetes when using all the host CPUs. The patch is done in the `kustomization.yaml` file.
```

Comment:
```suggestion
We added the `GOMAXPROCS=1` environment variable to limit the goroutines to 1 processor because node-exporter was getting hard throttled by Kubernetes when using all the host CPUs. The patch is done in the `kustomization.yaml` file.
```